simcon_project/templates/researcher_view.html

```diff
@@ -1,7 +1,7 @@
 {% extends "researcher_base.html" %}
 {% load render_table from django_tables2 %}
```

Comment:
please only git add files you have made code changes so we don't get confused/ break something